posix.mak

```diff
@@ -375,6 +383,16 @@ unittest/%.run : $(ROOT)/unittest/test_runner
 %.test : $(LIB)
 	$(MAKE) -f $(MAKEFILE) $(addsuffix .test,$(patsubst %.d,%,$(wildcard $*/*)))
 
+# Target for quickly debugging a single module
+# For example: DEBUGGER=ddd BUILD=debug make -f posix.mak std/format.debug
+# ddd in this case is a graphical frontend to gdb
+%.realdebug : %.d $(LIB)
```

Comment:
Nits:
- usually Makefile targets that are called before are put beforehand in the Makefile
- `.realdebug` sounds a bit weird, how about `.debug_with_debug_mode`, `.debug_internal`, or `debug2`?